reverse_iter_test.go

```diff
@@ -0,0 +1,224 @@
+package iradix
+
+import (
+	"sort"
+	"testing"
+	"testing/quick"
+)
+
+func TestReverseIterator_SeekReverseLowerBoundFuzz(t *testing.T) {
+	r := New()
+	set := []string{}
+
+	// This specifies a property where each call adds a new random key to the radix
+	// tree (with a null byte appended since our tree doesn't support one key
+	// being a prefix of another and treats null bytes specially).
+	//
+	// It also maintains a plain sorted list of the same set of keys and asserts
+	// that iterating from some random key to the end using LowerBound produces
+	// the same list as filtering all sorted keys that are lower.
```

Comment:
Is this comment right? Seems like copy paste from the normal direction perhaps we could change it to make it make sense for the reverse direction?

Reply:
Yes...again...thank you 😄